activities/tool-management/jitsi-guides.md

```diff
@@ -0,0 +1,45 @@
+---
+type: guide
+explains: account creation on our community instance of Jitsi Meet
+---
+
+# Jitsi Meet guides
```

Comment:
This new guide isn't linked from an index.

Reply:
Suggest changing the title to make more clear how it's different from https://about.publiccode.net/activities/tool-management/jitsi-meet.html (eg, guides for whom? why?)
Also suggest adding a link from the community Jitsi guide back to the main Jitsi tool page so that people signing up to community Jitsi know where to look for, for example, privacy info.